browser/ui/views/frame/brave_browser_view.cc

```diff
@@ -896,6 +899,11 @@ void BraveBrowserView::AddedToWidget() {
     GetBrowserViewLayout()->set_vertical_tab_strip_host(
         vertical_tab_strip_host_view_.get());
   }
+
+  if (secondary_contents_web_view_) {
```

Comment:
this needs browsers test to ensure that the url is properly updated when navigating and switching views

Reply:
Done. Created SplitViewLocationBarBrowserTest